Tryouts/Core/BasicModels/Modules/ProcessesViewModel.cs

```diff
@@ -0,0 +1,208 @@
+﻿/// ********************************************************************************************************
+///
+/// Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License").
+/// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+/// See the NOTICE file distributed with this work for additional information regarding copyright ownership.
+/// Unless required by applicable law or agreed to in writing, software distributed under the License
+/// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and limitations under the License.
+/// 
+/// ********************************************************************************************************
+
+using MorganStanley.ComposeUI.Tryouts.Core.Abstractions.Modules;
+using System.Collections.ObjectModel;
+using System.Reactive.Linq;
+using NP.Concepts.Behaviors;
+using NP.Utilities;
+
+namespace MorganStanley.ComposeUI.Tryouts.Core.BasicModels.Modules
+{
+    public class ProcessesViewModel
+    {
+        public ModuleViewModel[] Modules { get; }
+
+        IModuleLoader _moduleLoader;
+
+        public IEnumerable<SingleProcessViewModel> Processes { get; } =
+            new ObservableCollection<SingleProcessViewModel>();
+
+
+        public IEnumerable<SingleProcessViewModel> ProcessesWithWindows { get; } =
+            new ObservableCollection<SingleProcessViewModel>();
+
+        private IDisposable _modulesBehavior;
+        private IDisposable _processesBehavior;
```

Comment:
Who's responsible for disposing these IDisposables?

Reply:
They are not disposed, they stay there for the duration of the main process, to prevent the corresponding behaviors from disappearing due to a lack of reference.